lib/hermit-help

```diff
@@ -101,8 +101,15 @@ print_summary() {
   local summary usage help
   eval "$(documentation_for "$command")"
 
+  # Max number of characters we can print in summary without passing 80 total
+  summary_max=$(expr 80 - 9 - ${#command})
+
   if [ -n "$summary" ]; then
-    printf "   %-9s   %s\n" "$command" "$summary"
+    if [ "${#summary}" -gt "$summary_max" ]; then
+      printf "   %-20s   %s...\n" "hermit $command" "${summary:0:$summary_max}"
```

Comment:
I like the idea of wrapping the summaries, but I'd rather wrap them than truncate.  Maybe remove this change and we'll address it in a separate PR.